.gitignore

```
@@ -1,5 +1,5 @@
 .coverage
 data/
-.env
+/.env
```

This feels unsafe. What if we add a `.env` elsewhere that we don't want to commit, but we accidentally commit because it's no longer ignored? Instead, what about negating the pattern?

```suggestion
.env
!docker/.env
```